ucm2/sof-hda-dsp/HiFi.conf

```diff
@@ -14,7 +14,17 @@ SectionDevice."Mic1" {
 	Value {
 		CapturePriority 100
 		CapturePCM "hw:${CardId},6"
-		If.chn {
+		If.3mic {
+			Condition {
+				Type String
+				Haystack "${CardComponents}"
+				Needle "cfg-dmics:3"
+			}
+			True {
+				CaptureChannels 4
+			}
+		}
+		If.4mic {
```

Comment:
Use Regex condition here with cfg-dmics:[34] string here. It simplifies the config (keep one If).

Reply:
ack, done.